Makefile

```diff
@@ -131,7 +131,8 @@ test-coverage: node_version
 
 .PHONY: lint
 lint: node_version
-	npx eslint --cache --max-warnings 0 .
+	# max-warnings set to take account of legacy database migrations
+	npx eslint --cache --max-warnings 15 .
```

Comment:
revert in favour of eslint-disable directives (https://github.com/getodk/central-backend/pull/1371)